src/bridge/bridge.c

```diff
@@ -75,9 +75,12 @@ process_init (CockpitTransport *transport,
   gint64 version;
 
   if (!cockpit_json_get_int (options, "version", -1, &version))
-    version = -1;
```

Comment:
there are paths through `cockpit_json_get_int` that don't touch the value (`version` here).

Either we should return in this `if` block or set version to something invalid.